Apps/W1/Shopify/app/src/Products/Codeunits/ShpfyVariantAPI.Codeunit.al

```diff
@@ -140,8 +141,12 @@ codeunit 30189 "Shpfy Variant API"
         JResponse := CommunicationMgt.ExecuteGraphQL(GraphQuery.ToText());
         NewShopifyVariant := ShopifyVariant;
         NewShopifyVariant.Id := JsonHelper.GetValueAsBigInteger(JResponse, 'data.productVariantCreate.productVariant.legacyResourceId');
-        if NewShopifyVariant.Id > 0 then
+        if NewShopifyVariant.Id > 0 then begin
             NewShopifyVariant.Insert();
+            exit(true);
```

Comment:
Added a return value, as I want to mark the parent product with "Has Variants" only if a variant was successfully created.